package.json

```diff
@@ -9,7 +9,9 @@
     "main": "./lib/index.js",
```

Comment:
`description` should change to `UI Components for Visual Studio Code Extensions` to be consistent with the README.md title. And to comply with/avoid trademark usage.